api/docs/release.dox

```diff
@@ -282,6 +282,11 @@ Further non-compatibility-affecting changes include:
  - Added proc_get_vector_length_bytes() for AArch64. This returns the current
    vector length on all ARMv8 hardware including hardware which supports the
    Scalable Vector Extension (SVE).
+ - The addition of AArch64 SVE support required changes to dr_mcontext_t and
```

Comment:
This is in the wrong place: this list is non-compatibility changes.  This should go in the list of significant compatibility changes up above and should probably include the words "this is a binary compatibilty change".

Reply:
Moved to section titled: `The changes between version \DR_VERSION and 9.0.1 include the following compatibility changes` and re-worded to make the compatibility break more explicit.